plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoRestCatalog.java

```diff
@@ -471,7 +497,10 @@ public void createView(ConnectorSession session, SchemaTableName schemaViewName,
     @Override
     public void renameView(ConnectorSession session, SchemaTableName source, SchemaTableName target)
     {
-        restSessionCatalog.renameView(convert(session), toIdentifier(source), toIdentifier(target));
+        restSessionCatalog.renameView(convert(session), toRemoteView(session, source), toIdentifier(target));
```

Comment:
Could you add a test for renaming views?

Reply:
thanks, there was a miss that I caught while adding a test.